cmd/encryption-v1.go

```diff
@@ -367,7 +367,7 @@ func DecryptSource(client io.Writer, key []byte, metadata map[string]string) (io
 
 // DecryptRequest decrypts the object with the client provided key. It also removes
 // the client-side-encryption metadata from the object and sets the correct headers.
-func DecryptRequest(client io.Writer, r *http.Request, metadata map[string]string) (io.WriteCloser, error) {
+func DecryptRequest(client io.Writer, r *http.Request, seqNumber uint32, metadata map[string]string) (io.WriteCloser, error) {
```

Comment:
I guess it is much easier if we provide the wanted offset here, and then we internally calculate the sequence number & its relative offset, is it not possible with the current sio API ?

Reply:
No it is not possible right now.